app/models/post.rb

```diff
@@ -10,10 +10,15 @@ class Post < ApplicationRecord
 
   after_create do
     ActionCable.server.broadcast "posts_realtime", { row: PostsController.render(locals: {post: Post.last}, partial: 'post').html_safe }
+    ActionCable.server.broadcast "topbar", { review: Post.feedbacks_count }
   end
 
   after_create :autoflag
 
+  def self.feedbacks_count
+    Post.includes(:feedbacks).where( :feedbacks => { :post_id => nil }).count
```

Comment:
Okay, so some ActiveRecord magic in there manages to figure out what we want. It turns into a `LEFT OUTER JOIN`:

```
SELECT `posts`.`id` AS t0_r0, `posts`.`title` AS t0_r1, `posts`.`body` AS t0_r2, `posts`.`link` AS t0_r3, `posts`.`post_creation_date` AS t0_r4, `posts`.`created_at` AS t0_r5, `posts`.`updated_at` AS t0_r6, `posts`.`site_id` AS t0_r7, `posts`.`user_link` AS t0_r8, `posts`.`username` AS t0_r9, `posts`.`why` AS t0_r10, `posts`.`user_reputation` AS t0_r11, `posts`.`score` AS t0_r12, `posts`.`upvote_count` AS t0_r13, `posts`.`downvote_count` AS t0_r14, `posts`.`stack_exchange_user_id` AS t0_r15, `posts`.`is_tp` AS t0_r16, `posts`.`is_fp` AS t0_r17, `posts`.`is_naa` AS t0_r18, `posts`.`revision_count` AS t0_r19, `posts`.`deleted_at` AS t0_r20, `feedbacks`.`id` AS t1_r0, `feedbacks`.`message_link` AS t1_r1, `feedbacks`.`user_name` AS t1_r2, `feedbacks`.`user_link` AS t1_r3, `feedbacks`.`feedback_type` AS t1_r4, `feedbacks`.`post_id` AS t1_r5, `feedbacks`.`post_link` AS t1_r6, `feedbacks`.`user_id` AS t1_r7, `feedbacks`.`is_invalidated` AS t1_r8, `feedbacks`.`invalidated_by` AS t1_r9, `feedbacks`.`invalidated_at` AS t1_r10, `feedbacks`.`chat_user_id` AS t1_r11, `feedbacks`.`created_at` AS t1_r12, `feedbacks`.`updated_at` AS t1_r13, `feedbacks`.`is_ignored` AS t1_r14, `feedbacks`.`api_key_id` AS t1_r15, `feedbacks`.`chat_host` AS t1_r16 FROM `posts` LEFT OUTER JOIN `feedbacks` ON `feedbacks`.`post_id` = `posts`.`id` AND `feedbacks`.`is_invalidated` = 0 AND `feedbacks`.`is_ignored` = 0 WHERE `feedbacks`.`post_id` IS NULL
```